load/load_linux.go

```diff
@@ -16,6 +18,29 @@ func Avg() (*AvgStat, error) {
 }
 
 func AvgWithContext(ctx context.Context) (*AvgStat, error) {
+	if runtime.GOOS == "android" {
```

Comment:
You should just redefine the whole function body to your version I think, I don't see any point in keeping the old definition and doing a special case just for Android. It will be probably more optimized also.

Reply:
I found conflicting information online about how to translate the values into load floats. I think maybe it has changed over time, so it seems safer to only enable this for android right now (where load is broken altogether).

Reply:
The version in this PR I tested on modern linux kernels and confirmed the values are correct. I think it won't work on older kernels (whereas android always keeps relatively up to date kernels).

Reply:
Do you have specific kernel versions where it was said to be different? Go requires [2.6.23](https://github.com/golang/go/wiki/MinimumRequirements#linux), so maybe that's then always been calculated the same way starting from before this version.

Reply:
No I wasn't able to find the history. I only saw different stackoverflow solutions with different  advice.

I ended up doing the same thing as https://stackoverflow.com/a/4993273/332798 because that produced the correct results on kernels 4.9.113 and 5.4.15

But in [busybox](https://github.com/mirror/busybox/blob/b097a84d62dcfcf201a695b54bcfafae70cac8a6/procps/uptime.c#L67) and [here](https://stackoverflow.com/a/12303849/332798) they use `FSHIFT` also. When I tried that, the numbers produced were not correct.

Reply:
I'm not a fan about switching against runtime.GOOS, would prefer to call fileAvgWithContext, then sysinfoAvgWithContext in case of error returned by fileAvgWithContext.

Reply:
I don't think it is a good idea because it will break load average values on some systems without any error shown. But if you prefer I can change it.

Reply:
I misunderstood.. call fileAvg first. Okay, good idea!

Reply:
Done!